website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,114 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as running queries, loading or exporting data, and creating tables. While not necessary for basic operations, these options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Select your BigQuery project from the left-hand menu
+- Click **Settings** towards the top right
+- Scroll down to **Development Connection** and select BigQuery
+- Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configs you can set in dbt Cloud:
+
+#### Priority
+
+The `priority` for the BigQuery jobs that dbt executes can be configured with the `priority` configuration in your BigQuery profile. The priority field can be set to one of `batch` or `interactive`. For more information on query priority, consult the [BigQuery documentation](https://cloud.google.com/bigquery/docs/running-queries).
+
+#### Retries
+
+Retries in BigQuery help to ensure that jobs complete successfully by trying again after temporary failures, making your operations more robust and reliable.
+
+#### Location
+
+The `location` of BigQuery datasets can be set using the `location` setting in a BigQuery profile. As per the [BigQuery documentation](https://cloud.google.com/bigquery/docs/locations), `location` may be either a multi-regional location (e.g. `EU`, `US`), or a regional location (e.g. `us-west2`).
+
+You can specify the `location` parameter in your `dbt_project.yml` file or in your `models'` configuration. You can set the `location` to either a multi-regional location (e.g., `US`, `EU`) or a regional location such as `us-west2`.
+
+```yml
+
+models:
+  my_project:
+    +location: 'us-west2'  # Set the BigQuery dataset location
+
+```
+
+
+
+#### Maximum bytes build
+
+When a `maximum_bytes_billed` value is configured for a BigQuery profile, that allows you to limit how much data your query can process. It’s a safeguard to prevent your query from accidentally processing more data than you expect, which could lead to higher costs. Queries executed by dbt will fail if they exceed the configured maximum bytes threshhold. This configuration should be supplied as an integer number of bytes.
+
+```jinja
+
+my-profile:
```

Comment:
i think we chatted about this but did you mean to include this example here? it doesn't make sense to include a profiles.yml example in dbt cloud documentation